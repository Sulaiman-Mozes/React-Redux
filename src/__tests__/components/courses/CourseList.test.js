import React from 'react';
import { shallow } from 'enzyme';
import CourseList from '../../../components/courses/CourseList';
import { courses, authors } from '../../../../tools/mockData';


describe('CourseList Component', () => {
  it('should render the CourseList Component', () => {
    const props = {
      courses: courses.map(course => ({
        ...course,
        authorName: authors.find(a => a.id === course.authorId).name,
      })),
      onDelete: jest.fn(),
    };
    const wrapper = shallow(<CourseList {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
