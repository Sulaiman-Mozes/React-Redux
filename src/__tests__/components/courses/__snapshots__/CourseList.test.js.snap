// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CourseList Component should render the CourseList Component 1`] = `ShallowWrapper {}`;
